llpc/tool/llpcPipelineBuilder.h

```diff
@@ -72,6 +72,12 @@ class PipelineBuilder {
   // @returns : Calculated pipeline hash.
   virtual llvm::Error build() = 0;
 
+  // Output LLPC resulting binaries
+  //
+  // @param suppliedOutFile : Name of the file to output ELF binary
+  // @param firstInFile : Name of first input file
+  virtual llvm::Error outputElfs(const std::string &suppliedOutFile, llvm::StringRef firstInFile) = 0;
```

Comment:
Do we need the `firstInFile` argument? I think the implementation can chose it based on the input specs in compile info.

Reply:
Also, can we make `suppliedOutFile` a `StringRef`?